financial-connections/src/main/java/com/stripe/android/financialconnections/features/accountpicker/AccountPickerScreen.kt

```diff
@@ -143,6 +143,7 @@ private fun AccountPickerLoaded(
     LazyLayout(
         lazyListState = lazyListState,
         verticalArrangement = Arrangement.spacedBy(16.dp),
+        loading = payload is Loading,
```

Comment:
It’s currently only used in both account picker screens, as they are the only ones using the shimmer effect. Any other place where we should add it?

Reply:
I don't think so!

Reply:
maybe we want to be more specific with this field?